README.MD

```diff
@@ -21,8 +23,9 @@ Map<String, Object>  consumerConfig = ImmutableMap.<String, Object>builder()
 
 
 KafkaProtoParquetWriter<SampleMessage> writer =
-        new Builder<>("TestParquetWriter", "topic", consumerConfig, "/tmp/parquetFiles",
-                      SampleMessage.class, SampleMessage.PARSER)
+        new Builder<>("TestParquetWriter", "topic", consumerConfig, smartCommitOffsetTrackerPageSize,
+        smartCommitOffsetTrackerMaxOpenPagesPerPartition, SmartCommitMaxQueuedFiles, "/tmp/parquetFiles",
+        SampleMessage.class, SampleMessage.PARSER)
```

Comment:
I think it is better to define separate methods for setting mandatory parameters too (as we have for optional ones), because we have many parameters and passing all of them in builder constructor is not readable.